libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -90,6 +90,35 @@ public Optional<JourneyResponse> getJourneyResponseForStoredContraIndicators(
         return Optional.of(JOURNEY_RESPONSE_PYI_NO_MATCH);
     }
 
+    public boolean checkCiScoreBreaching(ContraIndicators contraIndicators)
+            throws UnrecognisedCiException {
+        LOGGER.info(
+                new StringMapMessage()
+                        .with(LOG_MESSAGE_DESCRIPTION.getFieldName(), "Retrieved user's CI items.")
+                        .with(
+                                LOG_NO_OF_CI_ITEMS.getFieldName(),
+                                contraIndicators.getContraIndicatorsMap().size()));
```

Comment:
This doesn't seem like the right place for this log message - this method doesn't actually retrieve the CIs!

I'd either delete it or move into the `CimitService` method (where we actually fetch the VCs).